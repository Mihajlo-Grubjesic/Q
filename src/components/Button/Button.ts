import styled from 'styled-components';

export const Button = styled.button`
  cursor: pointer;
  border: none;
  background-color: ${({ theme }) => theme.buttonBg};
  color: ${({ theme }) => theme.color};
  padding: 4px;
  border-radius: 4px;
  border: 1px solid ${({ theme }) => theme.color};
  font-size: 14px;
`;
