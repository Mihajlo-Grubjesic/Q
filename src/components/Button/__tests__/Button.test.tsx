import { render } from '@testing-library/react';
import { Button } from '../Button';

describe('Button', () => {
  it('renders button', () => {
    const { container } = render(<Button />);
    expect(container).toMatchSnapshot();
  });
});
