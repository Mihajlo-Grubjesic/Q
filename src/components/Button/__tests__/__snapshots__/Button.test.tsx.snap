// Vitest Snapshot v1

exports[`Button > renders button 1`] = `
<div>
  <button
    class="sc-bcXHqe fgvtsA"
  />
</div>
`;
